src/config/dynamic-theme-fixes.config

```diff
@@ -6533,6 +6533,13 @@ INVERT
 
 ================================
 
+hdlbits.01xz.net
+
+INVERT
+.image
```

Comment:
```suggestion
CSS
.image {
    background-color: white !important;
    filter: brightness(50%) sepia(40%) !important;
}
```

Reply:
This doesn't work well.

https://hdlbits.01xz.net/wiki/Fsm1

![image](https://user-images.githubusercontent.com/36977733/144426454-b21d1587-5396-40e6-9803-7d7f08f3b917.png)

Reply:
Hmm weird, seems like this has a `.image` wrapper and thus the class isn't on the `img` element(which it has to be in order to work), I think you can change it to:

`img.image, .image > img`

Reply:
It works well now, thanks!